latest/src/node_activation.rs

```diff
@@ -223,22 +228,27 @@ pub trait ContractStakeModule {
 
         // send unstake command to Auction SC
         let auction_contract_addr = self.settings().get_auction_contract_address();
-        let auction_contract = contract_proxy!(self, &auction_contract_addr, Auction);
+        let auction_proxy = contract_call!(self, auction_contract_addr, AuctionProxy);
         if unstake_tokens {
-            auction_contract.unStake(node_ids, bls_keys.into());
+            Ok(auction_proxy
+                .unStake(bls_keys.into())
+                .async_call()
+                .with_callback(self.callbacks().auction_unstake_callback(node_ids)))
         } else {
-            auction_contract.unStakeNodes(node_ids, bls_keys.into());
+            Ok(auction_proxy
+                .unStake(bls_keys.into())
```

Comment:
Changed from unStakeNodes to unStake. Intentional? If yes, then both branches are exactly the same.

Reply:
Merge error, GREAT catch!